src/components/edit-on-github-link/style.module.css

```diff
@@ -0,0 +1,20 @@
+.editLink {
+  align-items: center;
+  color: var(--blue-vibrant);
+  cursor: pointer;
+  display: inline-flex;
+  font-family: inherit;
+  font-size: 14px;
```

Comment:
I think there may be a point soon where we'll want to identify the type styles used in the Figma doc and create some abstraction for them. Maybe this is a `<Type />` component... or perhaps it's global utility classes as we're using in other projects.

The latter appeals to me because of simplicity, and because through all our time using global type utility classes, the only issue we've run into is nitpicking performance (which could be marginally better if we included ONLY the type classes used on a page, rather than all of them all the time). I think we'll have a lot hairier issues to tackle if we try to adopt a component pattern.

That being said, this is probably outside the scope of this PR - mostly just thinking out loud and putting this idea out in the universe 🎇

Reply:
Yes! As far as I know, there will be helper classes for different types coming from the design systems tokens. And they're supposed to match up with the types defined in Figma. They've been changing very frequently though, so I haven't created variables or classes at this point since even the names weren't fully worked out yet. I think we regroup on this after the holiday/first preview date and see if the naming of the tokens is more stable so we can start creating CSS variables at the very least?

Reply:
☝️ Sounds good! Yeah, I'm on board with regrouping on this after break 💯 

It seems like the structure at least is stable (eg some "body small" -ish style will exist, even if it's not called "body small"), so maybe we could even jump the gun a bit for the sake of our own developer experience in these initial phases and set up classes for those (and then run a global find-replace if the names change later). 

Also just noticed there's already an `npm` package published, version 0 i guess... definitely curious on how stable that feels since it seems that as we get into more varied UI work with stuff other than grays, it'd be really convenient to be able to pull in all the tokens at once via `npm` rather than having to recreate the CSS files in `dist` 🤔

Reply:
I agree it'd be more convenient to pull in all the tokens, but that's not what's been recommended by the design systems team. If you find that you have time to add some variables and CSS helper classes you need in passing I say go for it, but we're so constrained on time that I personally am not prioritizing that until we're past January 5th. I've avoided adding any extra variables that aren't in the designs I've worked with because the tokens aren't stable yet and don't want to lose too much time to creating and re-creating variables.